cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -33,13 +33,15 @@ if (NOT "$ENV{ARROW_BUILD_TOOLCHAIN}" STREQUAL "")
   set(JEMALLOC_HOME "$ENV{ARROW_BUILD_TOOLCHAIN}")
   set(LZ4_HOME "$ENV{ARROW_BUILD_TOOLCHAIN}")
   # orc disabled as it's not in conda-forge (but in Anaconda with an incompatible ABI)
-#   set(ORC_HOME "$ENV{ARROW_BUILD_TOOLCHAIN}")
+  # set(ORC_HOME "$ENV{ARROW_BUILD_TOOLCHAIN}")
   set(PROTOBUF_HOME "$ENV{ARROW_BUILD_TOOLCHAIN}")
   set(RAPIDJSON_HOME "$ENV{ARROW_BUILD_TOOLCHAIN}")
   set(SNAPPY_HOME "$ENV{ARROW_BUILD_TOOLCHAIN}")
   set(THRIFT_HOME "$ENV{ARROW_BUILD_TOOLCHAIN}")
   set(ZLIB_HOME "$ENV{ARROW_BUILD_TOOLCHAIN}")
   set(ZSTD_HOME "$ENV{ARROW_BUILD_TOOLCHAIN}")
+  # glog diabled as it is not compatible.
+  # set(GLOG_HOME "$ENV{ARROW_BUILD_TOOLCHAIN}")
```

Comment:
Did you run into an -fPIC issue? If so we should report it into conda-forge

Reply:
No. I'm trying to fix the Linux crash, it is not the reason.